doc/articles/guides/uno-material-walkthrough.md

```diff
@@ -0,0 +1,236 @@
+# How to use Uno.Material
+
+This guide will walk you through the necessary steps to setup and to use the [`Uno.Material` package](https://www.nuget.org/packages/Uno.Material) in an Uno Platform application.
+
+> [!TIP] The complete source code that goes along with this guide is available in the [unoplatform/Uno.Samples](https://github.com/unoplatform/Uno.Samples) GitHub repository - [UnoMaterialSample](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoMaterialSample)
+
+## Prerequisites
+
+### [Visual Studio for Windows](#tab/tabid-vswin)
```

Comment:
You should copy-paste the 'Prerequisites' section verbatim from the template. At the moment the rest of your article is inside the 'JetBrains Rider' tab, because you haven't closed the tab section with `***`

Reply:
Ah, My bad. I thought it was just "pure" markdown. Didn't know that there are docfx tabs and `[!NOTE]`s.